### DISCUSSION TITLE
Orage pi 5B audio missing / USB C cable

### DISCUSSION THREAD

**Discussion by enissayol** [Joshua-Riek/ubuntu-rockchip#183] [Q&A]
Hello,
I have an issue when I use the usb c cable to connect the orange pi 5B to an external monitor, I am not getting any sound from the monitor speakers. when I use the hdmi cable, the sound is working.
when i use the ubuntu version from Orange pi website, I have audio from the monitor speakers with the usb c cable.
Is there any fix for this issue ?

Thanks

**Comment by Joshua-Riek** [comment]
The Ubuntu version from Orange Pi uses a very weird sound driver configuration and has compatibility issues with some external devices. Thanks for bringing this problem to my attention. I have an idea where the issue might be, as I know audio over USB-C worked a while ago.

**Comment by enissayol** [comment]
Thanks for your reply, let me know if you find any fix, I don't want to install the Orange pi version of Ubuntu :), my setup is almost done, it's only this issue that is bothering me.

**Reply by Joshua-Riek** [reply]
Hey, i have been taking a look at the audio issue, and it broke in the v1.16 release. There were a lot of kernel changes and I will need to dig through it and figure out what broke the audio. Might take some time.

**Reply by enissayol** [reply]
Thanks for your reply, I've noticed one thing, when i remove pulseaudio and use only alsa, the sound is working but the issue is that i don't have any output device showing on the sound menu and i can't control the volume with the keyboard or the system.

**Reply by Joshua-Riek** [reply]
I may have found a fix to some of the USB-C display port issues. I'm running a build on my server now for further testing. But it should fix audio issues and cursor flickering when using the x11 display manager. https://github.com/Joshua-Riek/linux-rockchip/tree/patch-10

**Reply by Joshua-Riek** [reply]
@enissayol I finished my testing, and all the issues related to the USB-C display port should be fixed now. I need to wait on my kernel pull request to be merged, but the fix should be released sometime this coming week.

**Reply by enissayol** [reply]
Thank you very much! I'll wait for the update.
